lib/socket/socket.nit

```diff
@@ -147,6 +147,12 @@ class TCPStream
 		socket.write(msg.to_s)
 	end
 
+	redef fun write_byte(value)
+	do
+		if closed then return
```

Comment:
It applies the same behavior than the `write` method. But the whole module do need to be updated with better error support.